src/test/java/com/softserve/edu/greencity/ui/pages/tipstricks/TipsTricksPage.java

```diff
@@ -119,22 +107,31 @@ public boolean isDisplayedStartHabitBelow() {
     // Button 'Subscribe'
 
     public WebElement getSubscribeOnTipsTricks() {
-        return subscribeOnTipsTricks;
+        return subscribeOnTipsTricks = driver
+                .findElement(By.xpath("//div[@id='form-wrapper']/button[@class='primary-global-button']"));
     }
 
     public void clickSubscribeOnTipsTricks() {
         getSubscribeOnTipsTricks().click();
 
     }
 
+    private void waiting(int x) {
+        try {
+            Thread.sleep(x * 1000);
```

Comment:
please avoid using thread sleep in code

Reply:
Ok, thank you for your remark. 
I couldn`t use Explicite wait because when we navigate to TipsTricsPage there is a defect anchor of this page.